plugin.py

```diff
@@ -186,6 +208,9 @@ def on_pre_server_command(self, command: Mapping[str, Any], done_callback: Calla
                 return True
             elif command_name == "rust-analyzer.showReferences":
                 return self._handle_show_references(session, command, done_callback)
+            elif command_name == "rust-analyzer.triggerParameterHints":
+                done_callback()
```

Comment:
You could try something like `sublime.active_window().active_view().run_command('lsp_signature_help_show')` and see if that provides intended experience.

Reply:
Though maybe this already works as expected in ST, without explicitly triggering that command?